### PR TITLE
Learning from failures

### PR DESCRIPTION
**Pull request by jukecraft**

### CODE REVIEW

**Comment by SamirTalwar** [comment]
I think you need to double-space your paragraphs for Jekyll to render them correctly.

File: _posts/2015-07-14-learning-from-our-failures.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+tags:
+- failure
+- learning
+- github
+- travis
+- company culture
+---
+
+On my second day at Codurance, I managed to delete the repository with [Codurance's Github Pages](http://codurance.com/) on it. This made the website unavailable for the first time in a long time, and during best business hours.
+I passed by all the checks Github does to make sure you know what your doing, thinking I was about to delete the fork I accidentally created. When I saw afterwards that the fork was still there, I immediately realized what I had done.
```
Comment (SamirTalwar) [inline _posts/2015-07-14-learning-from-our-failures.md]: s/your/you're/

File: _posts/2015-07-14-learning-from-our-failures.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        href: https://flic.kr/p/88qwuz
+tags:
+- failure
+- learning
+- github
+- travis
+- company culture
+---
+
+On my second day at Codurance, I managed to delete the repository with [Codurance's Github Pages](http://codurance.com/) on it. This made the website unavailable for the first time in a long time, and during best business hours.
```
Comment (SamirTalwar) [inline _posts/2015-07-14-learning-from-our-failures.md]: Might be worth linking that to the actual repo rather than Codurance's home page.

Comment (SamirTalwar) [inline _posts/2015-07-14-learning-from-our-failures.md]: GitHub has a capital "H".

**Comment by SamirTalwar** [comment]
Aside from those minor issues, high five!

**Comment by jukecraft** [comment]
@SamirTalwar 
Yay!
I fixed the typos.
I was considering using the Link to the github pages, but then wondering if we want people to have a link to those GitHub pages? I know they can find it anyway, but wouldn't it be to obvious? I think the paragraphs are ok this way, do you agree?

**Pull request merged**